third_party/nvfuser/python_tests/test_python_frontend.py

```diff
@@ -743,6 +743,28 @@ def fusion_func(fd: FusionDefinition):
         test_fn(0)
         test_fn(1)
 
+    def test_index_select_scalar_indices(self):
```

Comment:
@naoyam tests added and verified the failing after reverting changes in `codegen.cpp`. It's all yours now.

Reply:
Oh, BTW, current branch I'm on is broken. You might want to revert the fouling commit for scatter 9340f80eab9393ea567fe18ad370bd6f31b8e062